### PR TITLE
Add property enable-full-list

### PR DESCRIPTION
**Pull request by wujack778**
- when enable-full-list is true, all group ids will  be listed.
- when enable-full-list is false, it is default situation.
 [issue 20218](https://github.com/Azure/azure-sdk-for-java/issues/20218)

### CODE REVIEW

**Comment by backwind1233** [comment]
@wujack778 resolve the conflicts, or just make a new PR

File: sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/aad/webapp/AADAccessTokenGroupRolesExtractionTest.java [hunk 1]
```diff
@@ -81,4 +82,11 @@ public void testGroupsNameAndGroupsId() {
         assertThat(groupsName).doesNotContain("ROLE_d07c0bd6-4aab-45ac-b87c-23e8d00194abaaa");
         assertThat(groupsName).hasSize(2);
     }
+
+    @Test
+    public void testEnableFullList() {
+        when(properties.getUserGroup().getEnableFullList()).thenReturn(true);
```
Comment (backwind1233) [inline sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/aad/webapp/AADAccessTokenGroupRolesExtractionTest.java:88]: can you write a unit test while getEnableFullList==false?

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/src/main/resources/application.yml [hunk 2]
```diff
@@ -6,6 +9,7 @@ azure:
     user-group:
       allowed-group-names: group1,group2
       allowed-group-ids: <group1-id>,<group2-id>
+      enable-full-list: false
     post-logout-redirect-uri: http://localhost:8080
```
Comment (backwind1233) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/src/main/resources/application.yml:13]: I think we should write some description about the param `enable-full-list` in readme.md

**Review by backwind1233 — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/src/main/resources/application.yml (hunk 2 above)
Comment (wujack778) [reply sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/src/main/resources/application.yml:13]: get it.

**Review by backwind1233 — APPROVED** [review]
LGTM

**Pull request merged**